Sources/NIO/EventLoopFuture.swift

```diff
@@ -322,7 +322,7 @@ public struct EventLoopPromise<T> {
 ///
 /// This means that for any `EventLoopFuture` that your code did not create itself (via
 /// `EventLoopPromise.futureResult`), use of `hopTo` is **strongly encouraged** to help guarantee thread-safety. It
-/// should only be elided when thread-safety is provably not needed.
+/// should only be elided when thread-safety is proved to be not needed.
```

Comment:
This doesn't need a change: [Merriam Webster](https://www.merriam-webster.com/dictionary/provably) defines "provably" as the adjectival form of "prove".